### ISSUE TITLE
Lack of table and wrong name

### ISSUE THREAD

**Issue by dilatchi** [azerothcore/tool-tc-migration#6]
Hello,

When you upload the tools and you finish the instructions you will get an error when you launch the authserver and worldserver
The solution is :

Rename the 7th column to "Token_key" in tables "account"
and upload [ip2nation](http://www.ip2nation.com/ip2nation/Download)
Now, you have your AUTH 

For the DB Characters, you have to add 3 tables
1) recovery_item
2) gameobject_respawn
3) creature_respawn

Now, you have your Characters DB

**Comment by FrancescoBorzi** [comment]
@junker1609 thanks a lot for this report. Since you know how to solve it already, can you send a PR with your solution?

**Comment by dilatchi** [comment]
> @junker1609 thanks a lot for this report. Since you know how to solve it already, can you send a PR with your solution?

Of course, you can check what I [changed](https://github.com/junker1609/tool-tc-migration/tree/patch-1)

**Comment by FrancescoBorzi** [comment]
@junker1609 it would be just easier to check if you open a PR via github :)

**Comment by Miorey** [comment]
@FrancescoBorzi @BarbzYHOOL  This pr https://github.com/azerothcore/tool-tc-migration/pull/8 solves this issue.

**Comment by dilatchi** [comment]
I sent a PR to fix this issue
